development/decision_process.rst

```diff
@@ -0,0 +1,175 @@
+
+RFCs are used to seek consensus on concrete plans that affect other parties.
+It is usually difficult to determine all the affected parties for published package interfaces. Changes to interfaces should thus typically go through this process.
+
+It's a good idea to contact any affected parties you *do* know about before starting this process to check that your resolution is sensible.
+Your institutional technical manager is always affected, as she or he is responsible for tracking your work schedule.
+If you are proposing work for others, they are obviously affected.
+Your institutional scientist, the DM System Architect (SA, K-T Lim), the DM Interface Scientist (IS, Gregory Dubois-Felsmann), and the DM PS (Mario Juric) are also valuable resources for determining affected parties.
+
+The purpose of an RFC is to inform others about the existence and content of the proposed decision and implementation in order to allow them to evaluate its impact, comment on it, refine it if necessary, and agree (implicitly or explicitly) or object (explicitly) to its execution.
+
+.. _decision-making-rfc-creating:
+
+Creating an RFC
+---------------
+
+Create an RFC by making a JIRA issue in the `Request For Comments (RFC) <https://jira.lsstcorp.org/projects/RFC>`_ project with a component of "DM."
+This will trigger postings to the `dm-devel mailing list <https://lists.lsst.org/mailman/listinfo/dm-devel>`_ and the `'Bot: RFC' HipChat room <hipchat://hipchat.com/room/1028779>`_.
+
+.. _decision-making-rfc-medium:
+
+RFC discussion medium
+---------------------
+
+The discussion of the RFC takes place in the medium of your choosing (typically this is JIRA RFC issue itself, but possibly `community.lsst.org <http://community.lsst.org/c/dm>`_, a HipChat room, a convened videocon, or an in-person meeting instead).
+You should be open to private communications as well.
+
+.. _decision-making-rfc-consensus:
+
+Role of the Assignee in reaching consensus
+------------------------------------------
+
+In our RFC process, the opinions of those who will be doing the work (and fixing any problems if something goes wrong) are given more weight.
+In some cases, this may mean that the RFC issue's Assignee passes to someone else.
+The opinions of more senior people or people more experienced in the area should also be given more weight and may also result in the Assignee changing.
+
+The Assignee is responsible for determining when no serious objections remain.
+In particular, there is no need to call for a formal vote on the (refined) resolution.
+If no explicit objections have been raised within, typically, 72 hours for "ordinary" issues and 1 week for "major" issues, the Assignee should assume that there are none.
+This is known as "lazy consensus."
+When this state has been reached, the Assignee is responsible for ensuring that the final consensus has been recorded in the RFC issue, marking it as **Adopted**, and proceeding with implementation of the decision.
```

Comment:
That is completely right, except you haven't put in that flow the "Attach implementation tickets" bit... (which is the bit I was talking about).